source/stylesheets/bootstrap/_navbar.scss

```diff
@@ -0,0 +1,87 @@
+.navbar.navbar-msm {
```

Comment:
I think we should move this to `/ui/components/_msm-navbar.scss` and just add classes that are independent of bootstrap. In you current implementation it is not apparent, which classes are new and which are bootstrap overrides.